source/Symbol/SwiftASTContext.cpp

```diff
@@ -1670,48 +1720,38 @@ lldb::TypeSystemSP SwiftASTContext::CreateInstance(lldb::LanguageType language,
           break;
       }
 
-      // First, prime the compiler with the options from the main executable:
-      bool read_options_from_ast = false;
-      ModuleSP exe_module_sp(target->GetExecutableModule());
-
-      // If we're debugging a testsuite, then treat the main test bundle as the
-      // executable.
-      if (exe_module_sp &&
-          PlatformDarwin::IsUnitTestExecutable(*exe_module_sp)) {
-        ModuleSP unit_test_module =
-            PlatformDarwin::GetUnitTestModule(target->GetImages());
+      ModuleSP exe_module_sp(GetModuleForCanonicalCompilerFlags(*target));
 
```

Comment:
Trivial, but exe_module_sp is no longer the correct name for this, since it isn't the main executable.  Maybe flags_module_sp?